apps/chat/src/components/Chat/TalkTo/ItemCardView.tsx

```diff
@@ -62,6 +63,7 @@ export const ItemCardView = ({
   disabled,
   isUnavailableModel,
   onSelectVersion,
+  hasContextMenu,
```

Comment:
```suggestion
  hasContextMenu = true,
```

guess should be like this?